src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -46,6 +46,11 @@ public DocumentDBConfiguration()
         /// </summary>
         public ChangeFeedHostOptions LeaseOptions { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ConnectionMode used in the DocumentClient instances.
+        /// </summary>
+        public ConnectionMode? ConnectionMode { get; set; }
```

Comment:
Another option is to remove the nullable here and explicitly choose the default (rather than letting the SDK choose it). Do you think that'd be more readable? It'd remove the requirement for checking HasValue, etc.

Reply:
I'd avoid having a default defined by us, the SDK already has a default value, and if that ever changes, we would be overriding the normal behavior?

Reply:
I was actually thinking we wanted to make sure the WebJobs Extension had the same default behavior, no matter what the Cosmos SDK did. But you're probably right, it's okay to do it this way. Updates to new versions will need to be explicit anyway in v2, so they'll need to consult default changes if there are any.